src/main/java/com/redhat/rhjmc/containerjfr/platform/internal/KubeEnvPlatformClient.java

```diff
@@ -76,7 +76,6 @@ private static ServiceRef envToServiceRef(Map.Entry<String, String> entry) {
             return null;
         }
         String alias = matcher.group(1).toLowerCase();
-        int port = Integer.parseInt(matcher.group(2));
-        return new ServiceRef(entry.getValue(), alias, port);
+        return new ServiceRef(entry.getValue(), alias);
```

Comment:
Same here - the form of these environment variables looks like `REDIS_MASTER_PORT_6379_TCP_ADDR=10.0.0.11` ( https://kubernetes.io/docs/concepts/services-networking/service/#environment-variables ) , so this will be a raw IP address.